styles/style.css

```diff
@@ -6,3 +6,110 @@ Green: #00B172
 White: #FFF
 
 */
+@import url('https://fonts.googleapis.com/css?family=Proxima+Nova');
+
+/*we tried to add the font Proxima Nova but assumed it can't be referenced.
+*/
+
+body {
+    color: #1A1A1A;
+    font-family: 'Proxima Nova';
+}
+
+.img-header {
+    width: 200px;
+}
+
+header {
+    display: flex;
+    justify-content: space-between;
+    padding: 0 20px;
+}
+
+nav ul {
+    list-style: none;
+    display: flex;
+    justify-content: space-between;
+}
+
+nav ul li {
+    padding: 0 7px;
+}
+
+.intro {
+    background-position: center;
+    background-image: url('/images/landing.jpg');
+    background-size: cover;
+    height: 100vh;
+    color: #FFF;
+    text-align: center;
+}
+
+.intro h1 {
+    padding-top: 350px;
+}
+
+.services h3 {
+    text-align: center;
+    text-decoration: underline #00B172;
```

Comment:
In this way the green divider takes all the screen width and the that's not our goal here.
You could wrap the h3 inside a span ` <h3> <span>What’s on Spotify?</span></h3>` and then use border-bottom: 
`.services h3 span {`
  `  border-bottom: 4px solid #00b172;`
`}`